pkg/minikube/bootstrapper/certs.go

```diff
@@ -117,9 +117,8 @@ func SetupCerts(cmd command.Runner, k8s config.KubernetesConfig, n config.Node)
 	}
 
 	for _, f := range copyableFiles {
-		glog.Infof("copying: %s/%s", f.GetTargetDir(), f.GetTargetName())
 		if err := cmd.Copy(f); err != nil {
-			return nil, errors.Wrapf(err, "Copy %s", f.GetAssetName())
+			return nil, errors.Wrapf(err, "Copy %s", f.GetSourcePath())
```

Comment:
not related to this PR but something to think about, what if the number of files to copy is more than 5, then make a tar out of them them copy one tar, and then untar them. (crazy idea for future) ?

Reply:
I've thought the same, but honestly, for the ssh runner, it wouldn't have much impact.

For kic_runner it would though.